test/testcafe/spec/ChallengeAuthenticatorPhone_spec.js

```diff
@@ -469,11 +469,14 @@ test
     await challengePhonePageObject.waitForErrorBox();
     await t.expect(challengePhonePageObject.getInvalidOTPFieldError()).contains('Invalid code. Try again.');
     await t.expect(challengePhonePageObject.getInvalidOTPError()).contains('We found some errors.');
+    await t.wait(30500);
```

Comment:
how is this time determined?

Reply:
this is hardcoded `resend timeout` 30s.. after which `resend warning` pops up.. If we can figure out some testcafe configuration to change this it would be great.

Reply:
Basically to for the test to see this warning it needs to wait.. because it is hardcoded in UI.

Reply:
Oh boy. How about skip testing the timeout and force render it?

Reply:
Or if you want to test the timeout, have it be configurable?

Reply:
yeah configuration for timeout make sense. but how do we do that ? I have filed a ticket for this issue earlier this week.